### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by LukeChung** [MicrosoftDocs/VBA-Docs#176]
Example should show how to create a report. Add a control, set some properties, save it to a specific name, etc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2ead3874-8f71-20b6-8bab-9b0648d6233e
* Version Independent ID: 0f5ece51-9a99-aea5-4fcd-4b41139ed700
* Content: [Application.CreateReport Method (Access)](https://docs.microsoft.com/en-us/office/vba/api/access.application.createreport#example)
* Content Source: [api/Access.Application.CreateReport.md](https://github.com/MicrosoftDocs/VBA-Docs/blob/master/api/Access.Application.CreateReport.md)
* Product: **access**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by barnam3** [comment]
Yes - the documentation could be a little more informative about proper and/or optional usage of this method. But any properties and/or associated methods should be limited to remarks only, to encourage exploring the documentation... i.e links to Application.CreateControl method or expression.[Report].[SomeReportProperty]. Show them where to look, but if this method cannot perform those actions, they shouldn't be in this method's example(s).
